client/my-sites/preview/main.jsx

```diff
@@ -66,23 +70,52 @@ class PreviewMain extends React.Component {
 	}
 
 	render() {
-		const { translate } = this.props;
+		const { translate, isPreviewable, site } = this.props;
+
+		if ( ! site ) {
+			// todo: some loading state?
+			return <span></span>;
```

Comment:
No loading state intended for this PR. Returning `null` caused an error so I did this